.github/workflows/linkcheck.yml

```diff
@@ -0,0 +1,51 @@
+name: Link Check
+run-name: Link Check
+
+permissions: read-all
+
+on:
+  workflow_dispatch:
+
+  pull_request:
+    branches: ['main']
+
+  # run weekly on Monday at 8am UTC
+  schedule:
+    - cron: '0 8 * * 1'
+
+env:
+  CORE: ./core
+  DOCS: ./docs
+
+jobs:
+  linkcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: defenseunicorns/pepr
+          path: ./core
```

Comment:
If the envar declared above ever changes, how will this string be kept up to date? Is that important? See the same for the `path: ./docs` a few lines down.